pages/instrumenting.md

```diff
@@ -58,12 +58,31 @@ information about the RPC. For instance when calling an HTTP service, providing
 the URI of the call will help with later analysis of requests coming into the
 service.
 
+The primary use case of binary annotations is exact match search. That said, it is
```

Comment:
Will reword this, though I wouldn't go so far as to say it is opposite even
if in your practice search isnt used.

On 12 Sep 2016 00:40, "Yuri Shkuro" <notifications@github.com> wrote:

In pages/instrumenting.md
<https://github.com/openzipkin/openzipkin.github.io/pull/35#discussion_r78298213>
:

> @@ -58,12 +58,31 @@ information about the RPC. For instance when calling an HTTP service, providing
>  the URI of the call will help with later analysis of requests coming into the
>  service.
>
> +The primary use case of binary annotations is exact match search. That said, it is

don't think this is accurate statement. I would even say the opposite, the
majority of binary annotations are never used for search, they are used to
add contextual data to the span.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/openzipkin/openzipkin.github.io/pull/35/files/eeb91ad188ac5257ec2123b9cf557674c74f3d7c#r78298213>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAD610FzBG_JGFGAQ42rhneJYKrnqyb9ks5qpC8NgaJpZM4IrHCO>
.